### DISCUSSION TITLE
v10.2.0-beta.1

### DISCUSSION THREAD

**Discussion by marcelklehr** [nextcloud/bookmarks#1766] [Announcements]
Hi everyone,

this is a beta release featuring some new.. features, that I hope you will be able to try out and provide feedback for on this thread.

What are the changes?
- UX: Directly crawl bookmark upon creation
  - [x] Needed testing: Performance of importing large number of bookmarks
- Support for Projects for bookmark folders and bookmarks
- Automatically create bookmarks from notes and link them via projects
  - [x] Needed testing: Does it work in all cases? Is this desirable? How could this be improved?
- Search AND conjunction
  - [x] Needed testing: Are the search results better now? Or worse?
- Implement bookmarks backup to Files
- Show and count duplicates
- Add explainers for SHARED_FOLDERS and UNAVAILABLE routes

/cc @sunjam

<hr /><em>This discussion was created from the release <a href='https://github.com/nextcloud/bookmarks/releases/tag/v10.2.0-beta.1'>v10.2.0-beta.1</a>.</em>

**Comment by marcelklehr** [comment]
@sunjam [commented](https://github.com/nextcloud/bookmarks/issues/447#issuecomment-1066158482)

> Note title can become title of the Bookmark

That might be confusing, if it's a large note with multiple links

> Note contents can become description for the Bookmark 

mmh, not sure, if we want to duplicate content without syncing it.

> convert system tag on a note to a bookmark tag

This could be done. Generally I'm hesitant to add tags automatically to things without the user's explicit consent, because it can be frustrating to have to clean up after a bot, if you want to use your own system. In this case, it would be the same tags the user has already created, though.... :thinking: I'm open to it.

**Reply by marcelklehr** [reply]
cc @nextcloud/designers @nextcloud/designlearners

**Comment by marcelklehr** [comment]
@jancborchardt says

> I’m not sure about the automatic creation of bookmarks – if we do, I’d say it’s better inside a "Notes" or ("Links from notes" folder inside the Bookmarks app (like we do for Talk media).
But if the link already exists in bookmarks, it could automatically get connected via these "related resources" indeed.

**Reply by marcelklehr** [reply]
I tend to agree and have implemented a dedicated folder where links from notes are put (existing links stay where they are).

**Comment by sunjam** [comment]
Very cool.  What do you mean by explainers?

**Reply by marcelklehr** [reply]
Short explanations in the UI on what each feature does.
